tests/performance-tests/src/tests/public-api/data-set-query/config.ts

```diff
@@ -0,0 +1,108 @@
+import { Options } from 'k6/options';
+import { QueryGeneratorConfig } from './queryGenerators';
+import {
+  dataSetQueryComparableOperators,
+  dataSetQueryIdOperators,
+} from '../../../utils/publicApiService';
+import steadyRequestRateProfile from '../../../configuration/steadyRequestRateProfile';
+import spikeProfile from '../../../configuration/spikeProfile';
+import rampingRequestRateProfile from '../../../configuration/rampingRequestRateProfile';
+import sequentialRequestProfile from '../../../configuration/sequentialRequestsProfile';
+import { parseIntOptional } from '../../../utils/utils';
+
+const profile = (__ENV.PROFILE ?? 'sequential') as
+  | 'load'
+  | 'spike'
+  | 'stress'
+  | 'sequential';
+
+const queries = (__ENV.QUERIES ?? 'simple') as 'simple' | 'complex';
+
+const limitToDataSetNames = __ENV.DATA_SET_NAMES?.split(',');
+const maxDataSetRows = parseIntOptional(__ENV.DATA_SET_MAX_ROWS) || undefined;
```

Comment:
`parseIntOptional` returns undefined so I don't think we need to specify `|| undefined`

Reply:
Oh yeah sorry - that's a hangover from before `parseIntOptional` was around.  Thanks, will sort.

Reply:
Done ta